Energy_generation_processing.py

```diff
@@ -0,0 +1,113 @@
+# %%
+import pandas as pd
+import numpy as np
+
+# %%
```

Comment:
same as below with using relative path, but it might be helpful to just have the below xls file in the repo so you dont need to reference out side of directory